analytics/management/commands/populate_analytics_db.py

```diff
@@ -116,16 +125,20 @@ def handle(self, *args: Any, **options: Any) -> None:
         stream.save(update_fields=["recipient"])
 
         # Subscribe shylock to the stream to avoid invariant failures.
-        # TODO: This should use subscribe_users_to_streams from populate_db.
-        subs = [
-            Subscription(
-                recipient=recipient,
-                user_profile=shylock,
-                is_user_active=shylock.is_active,
-                color=STREAM_ASSIGNMENT_COLORS[0],
-            ),
-        ]
-        Subscription.objects.bulk_create(subs)
+        Subscription.objects.create(
+            recipient=recipient,
+            user_profile=shylock,
+            is_user_active=shylock.is_active,
+            color=STREAM_ASSIGNMENT_COLORS[0],
+        )
+        RealmAuditLog.objects.create(
+            realm=realm,
+            modified_user=shylock,
+            modified_stream=stream,
+            event_last_message_id=0,
+            event_type=RealmAuditLog.SUBSCRIPTION_CREATED,
+            event_time=installation_time,
+        )
```

Comment:
I suppose this works, though sharing the code with `subscribe_users_to_streams` as suggested in the comment might be a better. (I think we probably would need to move this to `zilencer/`, but that's probably a correct thing to do anyway -- or really both should probably be moved to a new `development` app, but I'm not sure what that shouldbe called, and `zilencer/management/commands` is currently where we've been hiding from the self-hosted namespace devtool management command that might mess up your system if you run them)

Reply:
That seems like a very large can of worms for a couple of lines of not-overly-complicated shared code.

I'd very much love to not have `zilencer` be the dual purpose it is currently.

Reply:
Yeah let's start a #backend thread about splitting out a `development`; we can fiddle with this as a follow-up for that.

Reply:
[Done](https://chat.zulip.org/#narrow/stream/3-backend/topic/splitting.20out.20a.20development.20app/near/1557131).